.cirrus.yml

```diff
@@ -123,6 +126,64 @@ task:
       pg-ci-${CIRRUS_TASK_NAME}-${DATE}
 
 
+task:
+  matrix:
+    - name: openbsd-7-0
+      env:
+        PACKERFILE: packer/openbsd.pkr.hcl
+
+    - name: netbsd-9-2
+      env:
+        PACKERFILE: packer/netbsd.pkr.hcl
+
+  compute_engine_instance:
+    image_project: pg-ci-images-dev
+    image: debian-bullseye-nv
+    platform: linux
+    cpu: 4
+    memory: 16G
+    disk: 25
+    nested_virtualization: true
+    use_ssd: true
+
+
+  skip: $CIRRUS_LAST_GREEN_CHANGE != '' && $CIRRUS_CRON != 'regular-rebuild' && !changesInclude('.cirrus.yml', 'docker/linux_debian_packer', 'scripts/linux_debian_*', $PACKERFILE)
+  auto_cancellation: false
+
+  <<: *gcp_auth_unix
+
+  install_kernel_script: |
+    apt-get -y update
+    apt-get -y upgrade
+    apt-get -y dist-upgrade
+    apt-get install -y build-essential
+    apt-get install -y linux-headers-$(uname -r)
```

Comment:
I think this should do `export DEBIAN_FRONTEND=noninteractive` before, to get rid of some of the noisy output.